.travis.yml

```diff
@@ -3,6 +3,10 @@ python:
   - "2.7"
 node_js: 
   - "8"
+branches:
+  only:
+    - master
+    - develop
```

Comment:
Though it would make sense renaming  `develop` to `master` before merging this?

Reply:
It handles both. Later you can remove "develop" but it's not a big deal to keep it there forever.

Reply:
will this build two branches?

Reply:
no.